util/file.go

```diff
@@ -256,6 +256,23 @@ func JoinPath(elem ...string) string {
 	return filepath.ToSlash(filepath.Join(elem...))
 }
 
+// SplitPath splits the given path into a list, e.g. "/foo/bar/boo.txt" -> ["foo", "bar", "boo.txt"]
+func SplitPath(path string) []string {
+	var splitPath []string
+	dir, file := path, ""
+
+	for dir != "" {
+		dir, file = filepath.Split(filepath.Clean(dir))
+		splitPath = append([]string{file}, splitPath...)
+		if len(dir) == 1 && os.IsPathSeparator(uint8(dir[0])) {
```

Comment:
Ah, sorry for misreading `filepath.Split`. 

Quick sanity check: does this implementation behave differently than `strings.Split(filepath.Clean(path), filepath.Separator)`?

Reply:
Only slightly, strings.Split will return the root / as an empty string which is not a big deal :)

https://play.golang.org/p/asbyIPP-eiK